README.md

```diff
@@ -160,25 +162,25 @@ done
 This should produce output similar to the following:
 ```bash
 gpu-test1:
-pod1 ctr:
+pod0 ctr0:
```

Comment:
Since you changed the pod indexes to start from 0 - please, also change both of the `kubectl get pod -A` shell output quotes - they have pods starting with 1, and not having indexes at all.

Reply:
Done